src/main/java/no/nav/tpts/arena/ArenaSoapConfig.java

```diff
@@ -43,7 +46,31 @@ public YtelseskontraktV3 ytelseskontraktV3() {
                         .password(stsPassword)
                         .build())
                 .build();
-        
+
     }
 
+    @Bean
+    public TiltakOgAktivitetV1 tiltakOgAktivitetV1(
+            @Value("${app.arena.soap.sts.url}")
```

Comment:
Tror jeg ville hatt @Value på et private field (se lenger opp i fila). Om ikke annet for å øke lesbarhet, og for å slippe å ha @Value forskjellige steder i kode. Så slipper du også argumenter inn i metoden også.